python/LAMBDA_DLQ_CHECK/LAMBDA_DLQ_CHECK_test.py

```diff
@@ -0,0 +1,218 @@
+import sys
+import unittest
+
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import json
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::Lambda::Function'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+
+        raise Exception("Attempting to create an unknown client")
+
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('LAMBDA_DLQ_CHECK')
+
+
+class SampleTest(unittest.TestCase):
+    rule_empty_parameter_value = '{"dlqArn":""}'
```

Comment:
This is not possible. Config doesn't allow to pass an empty value in the lambda event.

Reply:
Done